README.md

```diff
@@ -378,6 +380,53 @@ sensu::agent::subscription { 'apache': }
 
 The resulting `agent.yml` would contain subscriptions for both `base` and `apache`.
 
+### Advanced agent - Annotations and Labels
+
+It's possible to define annotations and labels in many locations and the values merged into `agent.yml`:
+
+```puppet
+class { 'sensu::agent':
+  labels      => { 'foo' => 'bar', 'cpu.warning' => '70' },
```

Comment:
Could you please change this to use a better real world example instead of foo.

Reply:
Updated to use some useful examples